doc/operations/configuration.rst

```diff
@@ -0,0 +1,235 @@
+Configuration
```

Comment:
Comment: We've leaned hard into using `configuration` to describe sets of active nodes, and the adding/removing node protocols. This was fine while we started a node with a set of "options", but is confusing if we start a node with a "configuration". But it's going to confuse somebody that `configuration.rst` has nothing to do with `reconfiguration.rst` (or currently, `1tx-reconfig.rst` and `2tx-reconfig.rst`). In particular, if we dump a portion of this configuration file inside the KV, and think some of these settings are updateable by governance, what is _that_ called?

To be clear, I think this new use is sensible and absolutely the right term, matching common uses of the configure/configuration elsewhere, and I'm not suggesting a rename.

Reply:
Agreed. As discussed, there isn't an obvious alternative name here ("membership" is already used for consortium members) - perhaps "servicing a network" for what we currently call node reconfiguration? I'm happy to revisit this later and upgrade our docs once #2612 is done.